cron/healthToMongo.py

```diff
@@ -102,12 +77,6 @@ def getEsNodesStats():
     jsonobj = r.json()
     results = []
     for nodeid in jsonobj['nodes']:
-        # Skip non masters and non data nodes since it won't have full stats
```

Comment:
Do these type of nodes now contain full stats where we no longer have to do this logic?

Reply:
Yes, it's been doing this since the move to 5.x